### DISCUSSION TITLE
Export sav with encoding

### DISCUSSION THREAD

**Discussion by leontino** [Roche/pyreadstat#215] [Ideas]
I work with portuguese language, I to read with encoding='latin1' to read special characters, but when I export and read again, the special characters warp, I propose a idea to set encoding in export command.

**Comment by ofajardo** [comment]
It would be nice, unfortunately Readstat, the C library which is the backend of Pyreadstat does not support changing the encoding when writing, therefore it is not possible to implement. 

You can write an issue an [Readstat](https://github.com/WizardMac/ReadStat) directly asking the feature if you wish, it may take very long to be implemented if at all.

The default encoding when writing is utf-8, it definetly should support portuguese characters.

**Reply by leontino** [reply]
I'll try with him. Thanks

**Reply by ofajardo** [reply]
I have read your explanation on Readstat. Pyreadstat is asumming that when you write you use UTF-8, since that is what Readstat is expecting. What you can try is to read using latin1, then convert all strings in your dataframe to UTF-8, then write. That should fix your issue, i.e. you should be able to read the special characters correctly.
